build.gradle

```diff
@@ -44,8 +44,13 @@ dependencies {
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.17.0'
     implementation 'com.opencsv:opencsv:5.9'
 
-    implementation "org.codeforamerica.platform:form-flow:${formFlowLibraryVersion}"
-    println "📚Using form flow library ${formFlowLibraryVersion}"
+    if (profile == 'dev' || useLocalLibrary == 'true') {
+        implementation fileTree(dir: "$rootDir/../form-flow/build/libs", include: '*.jar')
+        println "📦 Using local library"
+    } else {
+        implementation "org.codeforamerica.platform:form-flow:${formFlowLibraryVersion}"
+        println "📚Using form flow library ${formFlowLibraryVersion}"
+    }
```

Comment:
I think this change is temporary and I will reverse this once the FFL changes have been released.